attach.go

```diff
@@ -17,6 +18,12 @@ import (
 func attach(sessionID string) {
 	dir := path.Join(threemuxDir, sessionID)
 
+	var shutdownMessage string
+	defer func() {
+		fmt.Print(shutdownMessage)
```

Comment:
Not sure if the shutdown message contains a plethora of useful info, but it might be useful to store it somewhere (like `/var/log/3mux/thedate`)

Reply:
>  but it might be useful to store it somewhere (like /var/log/3mux/thedate)

Oh, brilliant. I didn't think of using the timestamp; that's perfect. Since it would probably involve a UI change (showing the user in the session selector that a session failed), should I implement this logging in a separate PR?

Reply:
Sounds like a good idea!